bin/dredd-hooks-php

```diff
@@ -23,11 +23,48 @@ if (!$loaded) {
     );
 }
 
+// Get options from the command line
+$options = getopt('', [
+    'host:',
+    'port:',
+    'force',
+]);
+
+// Extract parameters from options
+if (array_key_exists('host', $options)) {
+	$host = $options['host'];
+}
+if (array_key_exists('port', $options)) {
+	$port = $options['port'];
+}
+$force = array_key_exists('force', $options);
+
+// Remove parameters from the list of arguments
+foreach($options as $key => $value) {
+	if ($pos = array_search(sprintf('--%s=%s', $key, $value), $argv)) {
+		unset($argv[$pos]);
+		continue;
+	}
+	$posKey = array_search(sprintf('--%s', $key), $argv);
+	$posValue = array_search($value, $argv);
+	unset($argv[$posKey]);
+	if ( 1 == abs($posKey - $posValue)) {
+		unset($argv[$posValue]);
+		continue;
+	}
+}
```

Comment:
@ddelnano For sure, it will be better and probably testable. I will look at it tonight.

Do you know how dredd starts the server? Does it append the hookfiles property after the language property? So if you happen to have
```yaml
# dredd.yml
hookfiles: /path/to/hooks
language: bin/dredd-hooks-php
```
does it trigger the server like?
```bash
bin/dredd-hooks-php /path/to/hooks
```
If so it would be a good idea to have the script running like to separate options from input
```bash
bin/dredd-hooks-php --host 0.0.0.0 --port 12345 --force -- /path/to/hooks
```